scripts/plugins.py

```diff
@@ -45,6 +48,8 @@
         Deploy_Plugins()
     elif mode == "install":
         Install_Plugins()
+    elif mode == "schema_parsing":
+        Get_Config_List(config_file, schema_file)
```

Comment:
Are you doing anything with this list? I believe this function returns the options and cli properties it parses.

Reply:
Not sure what this function does entirely. I was under the impression, it just exports all the configs into OS ENV variables. Correct me if this is NOT what this function does.

Reply:
I am trying to find a simple way to get this utility function to be used from the plugin. Also this is for one time use only as we will eventually move away from performing this functionality from plugins. If this is complex then I would not make any changes to plugin.py and use the helm plugin as is (which does parsing today)